boards/nxp/mimxrt700_evk/board.cmake

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright 2024 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if(CONFIG_SOC_MIMXRT798S_HIFI4)
+  board_runner_args(jlink "--device=MIMXRT798S_hifi4" "--reset-after-load")
+  board_runner_args(linkserver  "--device=MIMXRT798S:EVK-MIMXRT798")
```

Comment:
For linkserver do we need the `--core` argument, since this isn't the primary core?

Reply:
I haven't tested this yet and, since I'm not that familiar with i.MXRTs, I don't know exactly if this is needed or not. 
I can remove it, until I can test it. If needed, I'll add it later.